src/cmd/longevity_test/longevity.go

```diff
@@ -16,20 +16,25 @@ import (
 	"github.com/alcionai/corso/src/pkg/logger"
 	"github.com/alcionai/corso/src/pkg/path"
 	"github.com/alcionai/corso/src/pkg/repository"
+	"github.com/alcionai/corso/src/pkg/storage"
 	"github.com/alcionai/corso/src/pkg/store"
 )
 
 // deleteBackups connects to the repository and deletes all backups for
 // service that are at least deletionDays old. Returns the IDs of all backups
 // that were deleted.
+// Only supported for S3 repos currently.
 func deleteBackups(
 	ctx context.Context,
 	service path.ServiceType,
 	deletionDays int,
 ) ([]string, error) {
 	ctx = clues.Add(ctx, "cutoff_days", deletionDays)
 
-	r, _, _, _, err := utils.GetAccountAndConnect(ctx, service, nil)
+	r, _, _, _, err := utils.GetAccountAndConnect(ctx,
```

Comment:
nit: could you please put the `ctx` parameter on a newline as well?
```suggestion
	r, _, _, _, err := utils.GetAccountAndConnect(
		ctx,
```